stock_available_to_promise_release/models/stock_move.py

```diff
@@ -323,6 +323,8 @@ def _run_stock_rule(self):
                 new_move = move._release_split(remaining)
                 backorder_links[new_move.picking_id] = move.picking_id
 
+            move._before_release()
```

Comment:
As discussed and in a second improvement phase, investigate if this stock_available_to_promise_release could call again action_confirm on the updated move instead of manually creating a Procurement for the pull chain and run it (which is exactly what action_confirm does). If this is the case, then this new module would not require a glue module anymore and that would reduce the code and complexity

Reply:
Checked: `stock_available_to_promise_release` can't call `action_confirm` to do the work because

- releasing a move implies some split of qties
- `action_confirm` accepts only `draft` moves, and the one to release is already `confirmed`

So better to keep the current implementation.